__tests__/app.test.js

```diff
@@ -122,3 +125,56 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200: should return an an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        console.log(response.body);
+        expect(response.body).toBeInstanceOf(Array);
+        expect(response.body).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        response.body.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+        });
```

Comment:
you should be sending data back on a key that describes the data e.g. response.body.comments